### ISSUE TITLE
Добавить возможность запроса RSSI и MTU

### ISSUE THREAD

**Issue by AseevEIDev** [Monnoroch/blerpc-android#8]
Раз уж мы создаем библиотеку для всех и не предполагаем, что пользователям придется что-то доделывать под себя, то нужно предоставить максимальное количество действий которое может пригодиться. 

Одно из главных действий BLE - RSSI (received signal strength indicator), это запрос, который возвращает уровень сигнала, по которому мы можем с хорошей точностью определить, как далеко находится девайс.

Некоторые варианты применения:
- для нахождения девайса. Есть приложение для наушников Apple, которое показывает как далеко находится девайс (не показывает в какой стороне, но это легко определить пользователю)
- для навигации в помещении. https://habrahabr.ru/post/245325/

Чтобы получить RSSI нужно вызвать соответствующий запрос:
```
boolean isRequestSuccessful = bluetoothGatt.readRemoteRssi();
```

И обработать ответ в `BluetoothGattCallback`, который используется при создании `BluetoothGatt`:
```
@Override
public void onReadRemoteRssi(BluetoothGatt gatt, int rssi, int status) {
    workHandler.post(() -> {
        if (status != BluetoothGatt.GATT_SUCCESS) {
            handleRssiError("Read rssi failed, status=%d.", status);
        } else {
            handleRssiResult(rssi);
        }
    });
}
```

Также может понадобиться изменение размера MTU (Maximum Transmission Unit) - максимальный размер передаваемого пакета. 

Чтобы изменить MTU нужно вызвать соответствующий запрос:
```
boolean isRequestSuccessful = bluetoothGatt.requestMtu(mtuSize);
```

И обработать ответ в `BluetoothGattCallback`, который используется при создании `BluetoothGatt`:
```
@Override
public void onMtuChanged(BluetoothGatt gatt, int mtu, int status) {
    workHandler.post(() -> {
        if (status != BluetoothGatt.GATT_SUCCESS) {
            handleMtuError("Change mtu failed, status=%d.", status);
        } else {
            handleMtuResult(mtu);
        }
    });
}
```

Решение:
1) Учитывая, что запросы для получения RSSI и изменения MTU вызываются иным образом по сравнению с другими запросами, то нужно создать дополнительные типы запросов:
```
enum MethodType {
    UNKNOWN = 0;
    WRITE = 1;
    READ = 2;
    SUBSCRIBE = 3;
    RSSI = 4;
    MTU = 5;
}
```

И в `BleRpcChannel` обработать запросы аналогично `READ`, `WRITE`.

**Comment by Monnoroch** [comment]
Да, интересная мысль, спасибо. Предлагаю пока что держать идею в голове, но есть более важные к имплементации вещи.
